CHANGELOG.md

```diff
@@ -1,4 +1,100 @@
-<a name="9.10.0></a>
+<a name="10.0.0"></a>
+
+# [10.0.0](https://github.com/commercetools/ui-kit/compare/v9.10.0...10.0.0) (2019-06-20)
+
+### Breaking Changes
+
+This release introduces **breaking changes** which may entail some **migration steps**. We'll go through them now:
+
+- `hocs`: `withMouseOverState` and `withMouseDownState` have been dropped. Please use CSS properties instead. ([#881](https://github.com/commercetools/ui-kit/pull/881))
+
+- `Icons`: make icons themeable (breaking change) ([#726](https://github.com/commercetools/ui-kit/issues/726)) ([9bea3ad](https://github.com/commercetools/ui-kit/commit/9bea3ad))
+
+The prop `theme` has been renamed to `color` for Icons. Also, the values of this prop have changed. Please see the table below for the mapping between the old and new values.
+
+| old         | new       |
+| ----------- | --------- |
+| black       | solid     |
+| gray        | neutral60 |
+| white       | surface   |
+| blue        | info      |
+| green       | primary   |
+| green-light | primary40 |
+| orange      | warning   |
+| red         | error     |
+
+- `Text`: add `as` prop while deprecating `elementType` ([#631](https://github.com/commercetools/ui-kit/issues/631)) ([4bd6b42](https://github.com/commercetools/ui-kit/commit/4bd6b42))
+
+The prop `elementType` has been renamed to `elementType` for Text components. The `elementType` prop still exists, but is deprecated and will be removed in the next major UI-Kit release. Please migrate your components.
```

Comment:
Also:
> For `Text.Body` component, the `isInline` prop is also being deprecated. The same result is achieved by setting the `as` prop to `span`.

Reply:
I'll (try to remember to ) add this on Monday